queue/bolt_queue_storage_test.go

```diff
@@ -1,102 +1,78 @@
 package queue
 
 import (
-	"encoding/json"
-	"errors"
-
-	"github.com/coreos/bbolt"
-	"github.com/stretchr/testify/require"
+	"os"
+	"path/filepath"
 	"testing"
+	"time"
+
+	storm "github.com/asdine/storm"
+	require "github.com/stretchr/testify/require"
 )
 
-func TestBoltQueueStorage_PersistAndDeleteJob(t *testing.T) {
+func createStorm() *storm.DB {
+	dbPath, err := filepath.Abs(os.TempDir() + "/" + time.Now().String())
```

Comment:
`time.Now().String()` creates a filename containing spaces and other potentially problematic characters which cause issues and make windows builds panic, we should switch to `.UnixNano()` or similar

>dbPath, err := filepath.Abs(os.TempDir() + "/" + fmt.Sprint(time.Now().UnixNano()))